3rdparty/symcrypt_engine/CMakeLists.txt

```diff
@@ -87,11 +86,15 @@ if (UNIX)
       "/usr/lib/llvm-${CLANG_MAJOR_VERSION}/lib/clang/${CMAKE_C_COMPILER_VERSION}/include"
   )
 elseif (WIN32)
-  # Update the string when the supported clang version is changed
-  set(CLANG_VERSION "10.0.0")
-
-  set(CLANG_INTRINSIC_HEADERS_SEARCH_PATH
-      "C:/Program Files/LLVM/lib/clang/${CLANG_VERSION}/include")
+  find_program(
+    CLANG_COMPILER clang
+    PATHS "C:/Program Files/LLVM/bin"
+    NO_DEFAULT_PATH
+    REQUIRED)
+  get_filename_component(CLANG_COMPILER_ROOT_DIR ${CLANG_COMPILER} DIRECTORY)
+  file(GLOB 
```

Comment:
@CyanDevs  @asvrada  - Was this change done so this code is compiler version agnostic?

Reply:
Yes, the purpose of this change is so it would be able to support either clang 10 or 11 at the same time.